README.md

```diff
@@ -12,6 +12,7 @@ The following actions are supported:
 - Seek controls (<< and >>)
 - Mouse cursor movement
 - Left-mouse-button click using tapping
+- Right-mouse-button click using double tap
```

Comment:
Can you also please update the screenshot in this image to reflect the same?

Reply:
Will do :)